code/modules/surgery/organs/external/wings/functional_wings.dm

```diff
@@ -50,7 +50,7 @@
 
 ///Called on_life(). Handle flight code and check if we're still flying
 /obj/item/organ/external/wings/functional/proc/handle_flight(mob/living/carbon/human/human)
-	if(human.movement_type & ~FLYING)
+	if(!(human.movement_type & FLYING))
```

Comment:
I was about to correct you on this and then realized I'm the idiot that mixes up binary operators and added this bug in the first place